### PR TITLE
Make the generate validator use the test syntax

### PR DESCRIPTION
**Pull request by kevmoo**
Runs all generators, even if one fails
Updated travis.sh

### CODE REVIEW

**Comment by kevmoo** [comment]
@devoncarew will rebase on your last commit

File: tool/grind.dart [hunk 1]
```diff
@@ -59,72 +58,6 @@ void updateGhPages() {
     ..generate();
 }
 
-@Task('Run each generator and analyze the output')
-void test() {
```
Comment (devoncarew) [inline tool/grind.dart]: Let's keep this task; it can simplify to something like:

    new PubApp.local('test').run(['test/validate_templates.dart']);

Comment (kevmoo) [inline tool/grind.dart]: Done, although nothing prints until it finishes.

Is there a way to do async tasks in grinder?

**Comment by kevmoo** [comment]
@devoncarew PTAL

File: tool/grind.dart (hunk 1 above)
Comment (devoncarew) [inline tool/grind.dart]: There's an issue to track it: https://github.com/google/grinder.dart/issues/195.

**Comment by devoncarew** [comment]
lgtm!

**Pull request merged**